index.html

```diff
@@ -294,7 +294,7 @@
 
   ### <dfn>effectiveType</dfn> attribute
 
-  The `effectiveType` attribute, when getting, returns the <a>effective connection type</a> that is determined using a combination of recently observed round trip times and downlink bandwidth.
+  The `effectiveType` attribute, when getting, returns the <a>effective connection type</a> that is determined using a combination of recently observed <a href="#dom-networkinformation-rtt">rtt</a> and <a href="#dom-networkinformation-downlink">downlink</a> values.
```

Comment:
@marcoscaceres can't get the "plain" `<a>`'s to pick up the references. Any tips for how I can avoid hard-linking this?